crates/node/src/shell/prepare_proposal.rs

```diff
@@ -288,7 +289,10 @@ where
     tx.validate_tx().map_err(|_| ())?;
     if let TxType::Wrapper(wrapper) = tx.header().tx_type {
         // Check tx gas limit for tx size
-        let gas_limit = Gas::try_from(wrapper.gas_limit).map_err(|_| ())?;
+        let gas_scale = get_gas_scale(temp_state)
+            .expect("Failed to get gas scale from parameters");
```

Comment:
I see that we panic here while in `process_proposal` we raise an error. Can we use the same strategy in both cases?

Reply:
done in 037f62f73b6c7fd1e9a73b8d485a67caa8ec0767